components/List/src/MDCSelfSizingStereoCell.h

```diff
@@ -0,0 +1,65 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+#import "MDCBaseCell.h"
+
+/**
+ MDCSelfSizingStereoCell is intended to be an easy to use readymade implementation of a basic
+ "Stereo" cell. A stereo cell can be thought of as a roughly symmetrical cell with image views on
+ either side--like the speakers in an old fashioned boombox stereo.
+
+ The client is expected to configure the exposed subviews in any way they like from within
+ `-collectionView:cellForItemAtIndexPath:`. At the end of that method, the client should set the
+ cellWidth property so the cell has the information it needs to calculate the frames of its subviews
+ and its height.
```

Comment:
Docs need to be updated now that cellWidth is no longer relevant..

Instead though it might be helpful to include a small snippet showing how to set up fixed-width cells using estimated cell size.

Reply:
Done in 64652f0f58c0521445e92d50910e32fb9bdee360